scripts/typedoc/shopify-dev-renderer/getting-started.ts

```diff
@@ -43,7 +43,7 @@ export function gettingStarted(paths: Paths, options: Options = {}) {
 
   const docsInputPath = resolve(`${paths.inputRoot}/documentation`);
 
-  const files = ['extension-points.md'];
+  const files = ['extension-points.md', 'globals.md'];
```

Comment:
I wonder if we could support globbing somehow? Like, support adding all Markdown files via a string pattern like `'*.md'`. But, out of scope of this PR.

Reply:
Yeah would be a cleaner one